### PR TITLE
Judge count line graph

### LINKED ISSUE #132: write dashboard "judge count over appointment date line graph" script
## Task Description

- 

## User Stories

- As a user of the dashboard, I want to be able to see, for certain gender, verdict, etc. settings, when the relevant judges were appointed and see if there is a pattern to it.

## Relevant Files [If Available] 

-

### PR DESCRIPTION
**Pull request by luciegamby-sigmalabs**
Added judge count line graph. About to reserve merge conflicts.

Closes #132

### CODE REVIEW

**Comment by luciegamby-sigmalabs** [comment]
Just going to add some docstrings!

File: dashboard/charts.py [hunk 1]
```diff
@@ -58,6 +58,20 @@ def get_filtered_data(data: pd.DataFrame, filters: dict):
     return data
 
 
+def get_judge_appointed(conn: connect):
```
Comment (ayeshaa63) [inline dashboard/charts.py:61]: Return type hint?

File: dashboard/charts.py [hunk 2]
```diff
@@ -105,6 +119,25 @@ def get_waffle_chart(data: pd.DataFrame):
     return fig
 
 
+def get_judge_count_line_chart(conn):
```
Comment (ayeshaa63) [inline dashboard/charts.py:122]: return type hint?

File: dashboard/charts.py (hunk 1 above)
Comment (ayeshaa63) [inline dashboard/charts.py:61]: Could call it `get_judges_appointed` ?

**Review by ayeshaa63 — APPROVED** [review]
Great job!

**Pull request merged**